src/player/PlayerComponent.cpp

```diff
@@ -648,6 +652,50 @@ void PlayerComponent::handleMpvEvent(mpv_event *event)
       }
       break;
     }
+#if MPV_CLIENT_API_VERSION >= MPV_MAKE_VERSION(1, 100)
```

Comment:
Oh, and you'll want your commit message to have a top line like "player: Use new hooks API for libmpv >= 1.100; fixes #952"